### ISSUE TITLE
Mismatch between tags and latest version on PyPi (3.1.2)

### ISSUE THREAD

**Issue by thomasegriffith** [m-bain/whisperX#761]
The latest `whisperx` package on PyPi has a version of 3.1.2, but the latest tag in this repo is 3.1.1.

Any idea what commit the 3.1.2 package on PyPi is referencing?

Should there be a new 3.1.2. tag?

**Comment by erklem** [comment]
Linking some related issues. It seems the PyPi version is unofficial, but no GitHub whisperX maintainers have weighed in.

https://github.com/m-bain/whisperX/issues/700#issuecomment-1957790696
https://github.com/m-bain/whisperX/issues/721#issuecomment-2030084511
https://github.com/m-bain/whisperX/issues/722#issuecomment-2017297291
https://github.com/m-bain/whisperX/issues/859#issuecomment-2333034387

**Comment by erklem** [comment]
@daswer123 are you the PyPi maintainer for whisperX? If so, thank you for making the package easier to install!!

Could you help us answer this question - what causes the version differences between this GitHub repo and the PyPi package?

**Comment by daswer123** [comment]
Yes, I built this package on pypi, did it for personal purposes, if @m-bain asks I will transfer the rights, I specified that in the REAME on pypi.

I've included some recent PRs in there, and also added Align models for different languages.

Recently faster-whisper has been updated, so I recommend to fix faster-whisper to 1.0.1 along with the whisperx package.

**Comment by Barabazs** [comment]
> @erklem Yes, I built this package on pypi, did it for personal purposes, if @m-bain asks I will transfer the rights
> 
> I've included some recent PRs in there, and also added Align models for different languages.
> 
> Recently faster-whisper has been updated, so I recommend to fix faster-whisper to 1.0.1 along with the whisperx package.

Hi @daswer123 I'm currently helping Max out with project maintenance and we would very much appreciate the transfer of the whisperx namespace on PyPi. His PyPi username is `m-bain`.

**Comment by daswer123** [comment]
![image](https://github.com/user-attachments/assets/01d6a5c8-9720-4e6c-9e2f-0a94336ddbb4)

Done

**Comment by Barabazs** [comment]
Thank you @daswer123! Latest version will be uploaded asap
